apps/console/src/features/core/utils/event-publisher.ts

```diff
@@ -55,9 +55,9 @@ export class EventPublisher {
      * Function to publish event logs.
      * 
      * @param {string} eventId - Publishing event identifier.
-     * @param { {[key: string]: string} } [customProperties] - Any custom properties to be published (optional).
+     * @param { {[key: string]: any} } [customProperties] - Any custom properties to be published (optional).
```

Comment:
Using `any` is not a good practice Typescript. Can't we do the following.

```suggestion
     * @param { {[key: string]: string | Record<string, unknown>} } [customProperties] - Any custom properties to be published (optional).
```

Reply:
Only concern is this place as of now. `error` object can be of any type. All other places are strings
https://github.com/wso2/identity-apps/blob/master/apps/console/src/features/core/utils/http-utils.ts#L67

Reply:
`Record<string, unknown>` should support dynamic objects. It's similar to `object` type. We can't use `object` type due to it being flagged as an anti pattern in newer TS versions (https://github.com/typescript-eslint/typescript-eslint/issues/2068)

Reply:
fixed